pulp_rpm/app/settings.py

```diff
@@ -8,3 +8,4 @@
 INSTALLED_APPS = ["django_readonly_field", "dynaconf_merge"]
 ALLOW_AUTOMATIC_UNSAFE_ADVISORY_CONFLICT_RESOLUTION = False
 DEFAULT_ULN_SERVER_BASE_URL = "https://linux-update.oracle.com/"
+RPM_ITERATIVE_PARSING = True
```

Comment:
Is it worth documenting?
I'm trying to figure out when user would do that? In case there is some issue with parsing like we had, so they can switch and not break more data? If this is the case, then maybe it makes sense to keep it undocumented.

Reply:
>In case there is some issue with parsing like we had, so they can switch and not break more data? If this is the case, then maybe it makes sense to keep it undocumented.

That's exactly what I had in mind.  I don't think we should document it but the switch is there in case we need to flip it.